protocol.md

```diff
@@ -727,28 +727,26 @@ Upload-Length: 11
 Upload-Concat: final;/files/a /files/b
 ```
 
-### Client Tag
+### Upload Tag
 
 With this extension, Clients can provide a Tag for an upload - with which they
 can later retrieve the URL for the created resource.
 
-This allows Clients to resume an upload initiated through the [Creation With Upload](#creation-with-upload)
-or [Creation](#creation) extensions for which they did not receive the response with
-the newly created resource's URL (for example due to a broken connection).
+This allows Clients to resume an upload they initiated, but for which they did
+not receive the response with the newly created resource's URL (for example
+due to a broken connection).
 
-If the Server supports this extension, it MUST add `client-tag` to the `Tus-Extension`
+If the Server supports this extension, it MUST add `upload-tag` to the `Tus-Extension`
 header.
 
-To initiate an upload with Client Tag, Clients MUST include an `Upload-Tag`
+To initiate an upload with Upload Tag, Clients MUST include an `Upload-Tag`
 header with the initial `POST` request (as specified in the [Creation](#creation) and
 [Creation With Upload](#creation-with-upload) extensions) made to the Upload URL.
```

Comment:
We should probably also add a link to concatenation here just for consistency.

> as specified in the [Creation](#creation), [Creation With Upload](#creation-with-upload) and [Concatenation](#concatenation) extensions

Reply:
Good idea, done!